app/controllers/submissions_controller.rb

```diff
@@ -205,13 +205,13 @@ def collect_and_begin_grading
     grouping = Grouping.find(params[:id])
 
     if assignment.submission_rule.can_collect_grouping_now?(grouping)
-      # Push grouping to the priority queue
-      SubmissionCollector.instance.push_grouping_to_priority_queue(grouping)
+      grouping.update(is_collected: true)
+      grouping.save
       flash[:success] = I18n.t('collect_submissions.priority_given')
```

Comment:
This flash message can be deleted because we aren't doing anything about priority.